src/services/spotify/playlist/getPlaylistTracks.ts

```diff
@@ -32,6 +32,18 @@ export async function getSpotifyPlaylistTracks(
   });
   const json = await result.json();
 
+  if (json.next) {
+    const next = await getSpotifyPlaylistTracks({
+      token: args.token,
+      tokenType: args.tokenType,
+      playlistId: args.playlistId,
+      offset: args.offset + 100,
+      limit: 100
+    });
+
+    json.items = [...json.items, ...next.items];
+  }
+
   if (!result.ok) {

```

Comment:
This should be checked first, since it would indicate a non 2xx http response.

Reply:
Check that next.items is valid first you mean?

Reply:
The response from `fetch` will have an `ok` getter that is false if the http response wasn't 2xx.
So that indicates an error, and you won't have any of the expected results in the json object.

Reply:
Gotcha. Ill get something tomorrow for it.